pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+## Requirements
+
+- Filling out the template is required. Any pull request that does not include enough information to be efficiently reviewed may be rejected.
+- Please consider which branch this is to be submitted against. This will normally be the development branch, so please ask your reviewer if you think it needs to go somewhere else. 
+
+## Type
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Improvement to existing feature
+
+## Related JIRA Issue(s)
+
+_Please provide the URL(s) for any JIRA issues related to this PR._
+
+## Description
+
+_Using one or more sentences, describe the proposed changes and the reason for making them._
+
+## Views affected
+
+_List the website view(s) that you know are affected by this change._
+_If possible please provide a relative or localhost URL that can be used to view the change._
+
+### Other effects
+
+_List any other functionality that may be affected or which requires additional changes, such as saved configurations._ <!-- Please add an explanation if, for example, no test is needed -->
+
+- [ ] I have checked any requirements for Google Analytics
+- [ ] I have created any required tests
+
+## Possible complications
+
+_We appreciate this can be difficult but please highlight any views that you think might possibly be adversely affected by this change. For example a change to the services or app-level changes have potential for widespread consequences._
+
+## Merge conflicts
+
+_At certain stages of the release there is potential for many people to be working on the same modules. To avoid merge conflicts please confirm you have tested for this before submitting the pull request, eg by updating your feature branch and checking that the only changes being submitted are ones from you. <!-- Either check the box or describe any conflicts -->_
+
+- [ ] I confirm I have checked for merge conflicts
```

Comment:
If there are conflicts between the current branch and the destination branch, Github will block merging. I think it might also be possible to have our CI system immediately fail the CI step if it detects conflicts with the `dev` branch. Given this, what is the purpose of this section?

Reply:
We could use this section to add some information regarding any conflicts that occurred and the resolution that was made.

Reply:
But that should follow from the diff...
I wonder if any of you have seen a situation when people are reporting in their pull requests which files were conflicting and how the conflicts were resolved. I don't think I have.

Reply:
I've not seen any descriptions, but we have had examples where there have been conflicts that prevent merging (and more time going back). I would hope that this isn't an issue for us as a team so happy for it to not be included whilst this is for internal use. Can add it later if we need to